cfy-go/instances.go

```diff
@@ -217,39 +221,41 @@ func nodeInstancesOptions(args, options []string) int {
 	case "by-type":
 		{
 			operFlagSet := basicOptions("node-instances started")
-			var nodeType string
-			operFlagSet.StringVar(&nodeType, "node-type",
-				cloudify.KubernetesNode, "Filter by node type")
-
```

Comment:
Use: 
```
operFlagSet.StringVar(&nodeType, "node-type",
				os.Getenv("CFY_K8S_NODE_TYPE"), "Filter by node type")
```

Reply:
@0lvin check https://github.com/cloudify-incubator/cloudify-rest-go-client/tree/testing/cfy-go module it is up to date